### ISSUE TITLE
Unity page is unreachable

### ISSUE THREAD

**Issue by delapuente** [immersive-web/webvrrocks#294]
I'm missing something or I think this page https://webvr.rocks/unity is unreachable from the landing page.

**Comment by cvan** [comment]
yes, intentionally for now. I wanted to get basic pages stubbed out for the various tools first (see issue #199). but, at the very least, an announcement at the top plus links from the [Firefox page](https://webvr.rocks/firefox).

any ideas for a quick fix? I can do this shortly.

thanks for filing this issue; I've been meaning to.

**Comment by delapuente** [comment]
It makes sense right now since webvr.rocks does not include any listing of WebVR tools. If providing such information is one of the goals of the site, we can add it there.
